src/Payments/PaymentMethodRegistry.php

```diff
@@ -153,7 +153,11 @@ public function get_all_registered_script_data() {
 		$payment_methods = $this->get_all_active_registered();
 
 		foreach ( $payment_methods as $payment_method ) {
-			$script_data[ $payment_method->get_name() . '_data' ] = $payment_method->get_payment_method_data();
+			$script_data[ $payment_method->get_name() . '_data' ] =
+				array_merge(
+					$payment_method->get_payment_method_data(),
+					array( 'supports' => apply_filters( 'woocommerce_blocks_payment_gateway_features_list', $payment_method->get_supported_features(), $payment_method->get_name() ) )
```

Comment:
This is a nitpick I suppose but should this be prefixed as `__experimental_`?

Reply:
Noted.